tests/e2e/utils.js

```diff
@@ -11,8 +11,16 @@
 
 import { browser, expect, $ } from '@wdio/globals';
 
+export async function disableCache() {
+  if (browser.isChromium) {
+    await browser.sendCommand('Network.setCacheDisabled', {
```

Comment:
isn't there a way to disable network cache with Chrome command line arguments?

Reply:
No, there was in 2025, but they removed the flag.